.eslintrc.yml

```diff
@@ -20,13 +20,19 @@ rules:
   comma-dangle: [2, always-multiline]
   eqeqeq: [2, "allow-null"]
   no-shadow: 1
+  no-unused-vars: 1
+  no-console: 1
+  no-extra-semi: 2
+  no-useless-escape: 1
   quotes:
     - 2
     - single
     - allowTemplateLiterals: true
+      avoidEscape: true
```

Comment:
This avoids large git diff in tests.

Reply:
this should be moved to the test-only config file?

Reply:
done